packages/app/components/search/search-result.tsx

```diff
@@ -50,6 +54,20 @@ const SearchResult = ({
     return (
         <div className={styles.result}>
             <div>
+                {isLocalDocument && (
```

Comment:
checkbox for each documents.

Reply:
This works great—love it! My only concern is that we need to add an accessible name to the checkboxes below, because right now they're not associated with any name. Maybe we could add a "visually-hidden" label to associate with the inputs (probably using the document `localId` / `title`)? The click target is also pretty small: you have to click right on the checkbox, instead of like a checkbox with a label that lets you click on the label to check the checkbox. I'm not sure if we could make the label a bit larger with padding, then use a `span` inside to visually hide the text but still allow screen readers to correctly label the form element?

Reply:
Thanks for your feedback Ben...I was thinking about adding visually-hidden label to associate with the inputs. I will also modify the click target to look a little bigger.